docs/swagger.json

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
Maybe we could decide between one of the formats of the swagger file, that would also benefit us for not having to maintain two different files.